src/components/shared/ProtectedRoute.js

```diff
@@ -5,12 +5,16 @@ import { Route } from 'react-router-dom';
 import NotFound from '../NotFound/NotFound.react';
 
 const ProtectedRoute = props => {
-  const { accessToken, component: Component, ...restProps } = props;
+  const { accessToken, isAdmin, component: Component, ...restProps } = props;
   return (
     <Route
       {...restProps}
       render={routeProps =>
-        accessToken ? <Component {...routeProps} /> : <NotFound />
+        accessToken ? (
+          <Component {...routeProps} />
+        ) : (
+          <NotFound renderLogin={!isAdmin} />
```

Comment:
Shouldn't render login be true always?

Reply:
Handling the case when user is not admin and tries to access admin panel